code/modules/mob/living/basic/space_fauna/regal_rat/regal_rat_actions.dm

```diff
@@ -218,7 +36,7 @@
  * This action checks some nearby maintenance animals and makes them your minions.
  * If none are nearby, creates a new mouse.
  */
-/datum/action/cooldown/riot
+/datum/action/cooldown/mob_cooldown/riot
```

Comment:
Make sure these didn't gain a shared cooldown they didn't have before as a result of this repath
Might need to add `shared_cooldown = null`

Reply:
lol why were we treating this like a null value